Article/Article.js

```diff
@@ -3,24 +3,26 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.element.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerHTML = 'expand';
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle);
+
   }
 
   expandArticle() {
     // Using our reference to the article element, add or remove a class to expand or hide the article.
-
+    event.currentTarget.parentNode.classList.toggle('article-open');
   }
 }
 
 // START HERE: Select all classes named ".article" and assign that value to the articles variable
-let articles;
+let articles = document.querySelectorAll('.article');
 
 // Use .map() to iterate over the articles array and create a new instance of Article by passing in each article element as a parameter to the constructor.
-articles = Array.from(articles).map();
+articles = Array.from(articles).map((article) => {
+  new Article(article)})
```

Comment:
`map` expects a return, so at this point in the code, articles is equal to an array containing nothing but `undefined` elements